### PR TITLE
Replaced usings of GetObjectID method with GetObjectNo

### PR DESCRIPTION
**Pull request by un-tone**
For https://github.com/Starcounter/level1/pull/4545
GetObjectID removed from Starcounter API

P.S. I'm not sure who the best person for reviewing this PR, so feel free to remove yourself from the reviewers if you want.

### CODE REVIEW

**Comment by Mihaiii** [comment]
Regarding the nuget package, should we bump major after this PR is merged?

**Comment by un-tone** [comment]
@habib535 agree. In the tomorrow SC build, this and the related methods will be already deprecated.
There is enough time for other apps updating before we will merge this and completely remove that methods.

File: Ring1/Something.cs [hunk 1]
```diff
@@ -23,7 +23,7 @@ public string Key
         {
             get
             {
-                return this.GetObjectID();
+                return this.GetObjectNo().ToString();
```
Comment (warpech) [inline Ring1/Something.cs]: Does `GetObjectNo().ToString()` return base64 or base10 encoded representation of the `GetObjectNo`? It is not clear to me at all. Please clarify that in docs and intelisense.

Comment (un-tone) [reply Ring1/Something.cs]: No, it just returns a string representation of the `ObjectNo` field. I guess there is nothing to clarify in docs.

**Comment by un-tone** [comment]
The changes are canceled. Look at https://github.com/Starcounter/level1/issues/4210#issuecomment-373794845

**Pull request closed without merging**